### PR TITLE
Update upgrade-distribution-lists.md

### PR DESCRIPTION
**Pull request by msbemba**
removed Escape characters before angle-brackets

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5083

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/upgrade-distribution-lists.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ If you're experienced at using PowerShell, you might want to go this route inste
 To upgrade a single DL, run the following command:
 
 ```PowerShell
-Upgrade-DistributionGroup -DlIdentities \<Dl SMTP address\>
+Upgrade-DistributionGroup -DlIdentities <Dl SMTP address>
 ```
 
 For example, if you want to upgrade a DLs with SMTP address dl1@contoso.com, run the following command:
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/upgrade-distribution-lists.md]:
```suggestion
For example, if you want to upgrade a DL with SMTP address dl1@contoso.com, run the following command:
```

File: microsoft-365/admin/manage/upgrade-distribution-lists.md [hunk 2]
```diff
@@ -98,8 +98,8 @@ Upgrade-DistributionGroup -DlIdentities dl1@contoso.com
 You can also pass multiple DLs as a batch and upgrade them together:
 
 ```PowerShell
-Upgrade-DistributionGroup -DlIdentities \<DL SMTP address1\>, \< DL SMTP address2\>,
-\< DL SMTP address3\>, \< DL SMTP address 4\>
+Upgrade-DistributionGroup -DlIdentities <DL SMTP address1>, < DL SMTP address2>,
+< DL SMTP address3>, < DL SMTP address 4>
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/upgrade-distribution-lists.md]: The spacing does not seem right. Should there be a space after the first angle in both instances? And also, "address3" but "address 4". Should there be a space between "address" and the numeral?

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/manage/upgrade-distribution-lists.md (hunk 2 above)
Comment (msbemba) [reply microsoft-365/admin/manage/upgrade-distribution-lists.md]: I'll fix that

Comment (msbemba) [reply microsoft-365/admin/manage/upgrade-distribution-lists.md]: It has been updated now.

**Review by JohanFreelancer9 — APPROVED** [review]
@msbemba, thank you.

**Pull request merged**